designTokens/size.ts

```diff
@@ -81,6 +81,40 @@ export const size: DesignTokens = {
   },
   form: {
     element: {
+      s: {
+        min: {
```

Comment:
could you maybe in the same also create the token for the form element width default? (300px), it's currently missing (and should be used in form field instead of defining there.

Reply:
Should I add `width`, or both `width` / `min-width`?

![image](https://github.com/lyne-design-system/lyne-design-tokens/assets/101575400/ab3ad0d0-29a4-4718-a04f-85dc511fcea3)